client-v2/src/types/Capi.ts

```diff
@@ -98,6 +98,7 @@ interface CapiArticleFields {
   liveBloggingNow?: CapiBool;
   shortUrl?: string;
   membershipUrl?: string;
+  newspaperEditionDate?: string;
```

Comment:
could this be a `Date` type?

Reply:
After a chat - I'll add a new field to `frontsMeta` which flags `isPrintSent` to allow us to display draft content as if it was live in the editions version of the tool.